samples/matter/common/src/led_widget.cpp

```diff
@@ -6,14 +6,18 @@
 
 #include "led_widget.h"
 
+#ifdef CONFIG_DK_LIBRARY
```

Comment:
I think there might be no point to compile this file if the mentioned Kconfig is not set :)

Reply:
Yes, it can try to use those methods even if the CONFIG_DK_LIBRARY is not set. We can do it as Kamil wrote as a follow-up to the task regarding refactoring the structure of the common files.